src/importexport/musicxml/internal/musicxml/exportxml.cpp

```diff
@@ -4162,13 +4162,19 @@ static void writePitch(XmlWriter& xml, const Note* const note, const bool useDru
         default:                                             break;
         }
     }
-    if (alter && !alter2) {
-        xml.tag("alter", alter);
+    // Override accidental with explicit note tuning
+    double tuning = note->tuning();
+    if (!muse::RealIsNull(tuning)) {
+        alter2 = tuning / 100.0;
+    }
+    // `alter` represents the "regular" (Western) pitch which can be
```

Comment:
We don't need the comment - renaming `alter2` to something like `microtonalAlter` would make this clear.

Reply:
Done